src/components/Person.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from 'prop-types';
+
+const Person = ({ name, age, sex, isMarried, partnerName }) => (
+    <>
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age && (
+        <p className="Person__age">I am {age}</p>
+      )}
+      {(isMarried && sex === 'm') && (
+        <p className="Person__partner">My wife&apos;s name is {partnerName}</p>
+      )}
+      {(isMarried && sex === 'f') && (
+        <p className="Person__partner">My husband&apos;s name is {partnerName}</p>
+      )}
+    </section>
+    </>
+)
+
+Person.defaultTypes = {
```

Comment:
You should add defaultProps only for non required props
```suggestion
Person.defaultProps = {
```